src/Analyzers/Analyzers/src/Microsoft.AspNetCore.Analyzers.csproj

```diff
@@ -10,7 +10,7 @@
     -->
     <IsShippingPackage>false</IsShippingPackage>
 
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>$(DefaultNetFxTargetFramework);netstandard2.0</TargetFrameworks>
```

Comment:
Not sure whether adding a .NET Framework assembly for analyzers is useful. @pranavkm❔

Reply:
Yeah, I don't think this should be necessary. The ns2.0 version works in VS and CLI, so unless something's broken there, I wouldn't change these.